camera_index: 0
frame_width: 640
frame_height: 480
